### PR TITLE
Image Settings for Porter Apps V1

### PR DESCRIPTION
**Pull request by ferozemohideen**

### CODE REVIEW

File: dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx [hunk 1]
```diff
@@ -36,42 +36,41 @@ const ImageSettingsTab: React.FC = () => {
 
     return (
         <>
-            {match(source)
-                .with({ type: "docker-registry" }, (source) => (
```
Comment (ianedwards) [inline dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx:40]: guessing we make this check elsewhere?

File: dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx [hunk 2]
```diff
@@ -36,42 +36,41 @@ const ImageSettingsTab: React.FC = () => {
...
+            <Spacer y={1} />
+            <Button
+                type="submit"
+                status={buttonStatus}
+                disabled={
+                    isSubmitting
+                    || latestRevision.status === "CREATED"
+                    || latestRevision.status === "AWAITING_BUILD_ARTIFACT"
+                    || !image.repository
+                    || !image.tag
```
Comment (ianedwards) [inline dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx:56]: is image always defined?

Comment (ianedwards) [reply dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx:56]: either something I misconfigured or a bug in react-hook-form - I think `watch` return values mistakenly say they're defined when potentially they're not. Not sure if that's the case here

**Review by ianedwards — APPROVED** [review]

File: dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx (hunk 1 above)
Comment (ferozemohideen) [reply dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx:40]: decided to add this back

File: dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx (hunk 2 above)
Comment (ferozemohideen) [reply dashboard/src/main/home/app-dashboard/app-view/tabs/ImageSettingsTab.tsx:56]: added conditional

**Pull request merged**